ChangeLog

```diff
@@ -1,15 +1,17 @@
-------------------
 Pylint's ChangeLog
```

Comment:
Unrelated change to the formatting of the changelog should be reverted. The link fix can be added in another merge request.